rules/TypeDeclaration/Rector/ClassMethod/StrictArrayParamDimFetchRector.php

```diff
@@ -167,6 +171,10 @@ private function shouldStop(Node $node, string $paramName): bool
             $nodeToCheck = $node->var;
         }
 
+        if ($nodeToCheck instanceof Node\Expr\MethodCall && $this->isName($nodeToCheck->var, $paramName)) {
+            return true;
```

Comment:
`$nodeToCheck->var` can be not variable, so better to check `$nodeToCheck->var` instanceof `Node\Expr\Variable` too

Reply:
:+1: